### PR TITLE
Task/adev 50 create env specific sorting replicas fix env

### PR DESCRIPTION
**Pull request by jsoneaday**
fix issue where docker does not see ENV variable.

### CODE REVIEW

File: .github/workflows/prod.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ on:
   push:
     tags:
     - "*"
+env:
```
Comment (macku99) [inline .github/workflows/prod.yml:7]: this is still too early I think...

File: .github/workflows/staging.yml [hunk 2]
```diff
@@ -4,7 +4,9 @@ on:
   push:
     branches:
     - 'feature/rebranding'
-
+env:
```
Comment (macku99) [inline .github/workflows/staging.yml:7]: same here

File: .github/workflows/prod.yml [hunk 3]
```diff
@@ -69,6 +72,7 @@ jobs:
           sed -i -e "s|ALGOLIA_SEARCH_API_KEY_B64|$(echo -n ${{ secrets.ALGOLIA_SEARCH_API_KEY }}| base64)|g" ci/prod.yaml
           sed -i -e "s|ALGOLIA_WRITE_API_KEY_B64|$(echo -n ${{ secrets.ALGOLIA_WRITE_API_KEY }}| base64)|g" ci/prod.yaml
           sed -i -e "s|ALGOLIA_INDEX_B64|$(echo -n ${{ secrets.ALGOLIA_INDEX }}| base64)|g" ci/prod.yaml
+          sed -i -e "s|ENV_B64|$(echo -n ${{ vars.ENV }}| base64)|g" ci/staging.yaml
```
Comment (macku99) [inline .github/workflows/prod.yml:75]: this should be ci/prod.yml

File: .github/workflows/prod.yml (hunk 1 above)
Comment (jsoneaday) [reply .github/workflows/prod.yml]: Removed

File: .github/workflows/staging.yml (hunk 2 above)
Comment (jsoneaday) [reply .github/workflows/staging.yml]: Removed

File: .github/workflows/prod.yml (hunk 3 above)
Comment (jsoneaday) [reply .github/workflows/prod.yml]: Updated as prod. Also fixed ci/staging.yaml variable name

**Review by macku99 — APPROVED** [review]
LGTM

**Pull request merged**